bedrock/mozorg/templates/mozorg/contribute/thankyou.html

```diff
@@ -43,21 +43,50 @@ <h1 class="page-title">{{ _('Thanks!') }}</h1>
     <div class="container">
       <div class="confirm-body">
 
-        <div class="mozillians-profile">
-          <span class="profile-greeting">{{ _('Hello!') }}</span>
-          <span class="profile-name">{{ _('Your Name') }}</span>
-        </div>
-        <h2 class="section-title">{{ _('In the meantime&hellip;') }}</h2>
-        <p class="section-tagline">{{ _('You can create your Mozillians account now to connect with fellow Mozillians.' ) }}</p>
+      {% if (category == 'coding') or (category == 'testing') or (category == 'helping') or (category == 'translating') %}
+        <div class="mozillians-profile oneandone"></div>
+        <h2 class="section-title">
+          {{ _('In the meantime&hellip;') }}</h2>
+        <p class="section-tagline">
+          {% if l10n_has_tag('contribute_cta_updates') %}
+            {{ _('You can help by picking an easy task that only takes a few minutes - or take on a bigger challenge.') }}
+          {% else %}
+            {{ _('You can create your Mozillians account now to connect with fellow Mozillians.' ) }}
+          {% endif %}
+        </p>
         <p class="cta-mozillians">
-          {% if settings.DEV %}
-          <a href="https://mozillians-dev.allizom.org/?source=contribute&amp;utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=contribute-page">
+          <a href="https://oneanddone.mozilla.org?source=contribute&amp;utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=contribute-page">
+          {% if l10n_has_tag('contribute_cta_updates') %}
+            {{ _('Choose a Task Now') }}
           {% else %}
-          <a href="https://mozillians.org/?source=contribute&amp;utm_source=mozilla.org&amp;utm_medium=referral&amp;utm_campaign=contribute-page">
+            {{ _('Yes, create my Mozillians account') }} *
           {% endif %}
+          </a>
+        </p>
+        
+      {% elif (category == 'writing') or (category == 'teaching') or (category == 'activism') or (category == 'dontknow') %}
```

Comment:
Hm, I think we could be safe with a simple `else` here (instead of specifying categories). Basically the `if` handles special cases (which we could add to in the future if needed), and the `else` would handle, well, everything else. :grin: 

The bug mentions sending to https://www.mozilla.org/contribute/friends/ as a fallback, so it should be safe to use an `else` default instead of a specific `elif`.